src/include/duckdb/common/vector_operations/generic_executor.hpp

```diff
@@ -31,18 +31,32 @@ struct PrimitiveType {
 	}
 
 	INPUT_TYPE val;
+	bool is_null;
```

Comment:
Can we have a `ExecutorBaseType` class that has this member that all types inherits from? Every `*Type` class needs this field